static/js/main.js

```diff
@@ -204,7 +209,24 @@ var ep = window['ep'] || {
     },
 
     _markUsedBy: function(slot_id, username) {
-        // TODO: add GUI
+        var lock_container = $("#module_header_"+slot_id).find(".module_lock");
+        if (username) {
+            lock_container.text(ep._username_alias);
+        } else {
+            //Module is not locked by anyone
+            lock_container.empty();
+        }
+    },
+
+    parseWSMessage: function (message) {
```

Comment:
None of the existing methods do a safety check on slot_id. If you think it's necessary in this case, we should do it in python code of handlers